src/main/java/com/ldtteam/structurize/util/LanguageHandler.java

```diff
@@ -235,7 +236,8 @@ private void load(final String path, final boolean generatedDefault)
             InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream(String.format(path, locale));
             if (is == null)
             {
-                is = Thread.currentThread().getContextClassLoader().getResourceAsStream(String.format((generatedDefault ? "../generated/resources/" : "") + path, defaultLocale));
+                is = Thread.currentThread().getContextClassLoader().getResourceAsStream(
+                  String.format((generatedDefault && !FMLEnvironment.production ? "../generated/resources/" : "") + path, defaultLocale));
```

Comment:
You could just do at line 219 load(fileLoc, !FMLEnvironment.production)

Reply:
Try to make it so it works automatically without the generatedDefault boolean flag

Reply:
I was just trying to account for the possibility that minecolonies might want to also use genned lang sometime.

Reply:
But it still would be based on in which space you are